rules/manager.go

```diff
@@ -423,12 +432,32 @@ func (g *Group) run(ctx context.Context) {
 					g.metrics.IterationsScheduled.WithLabelValues(GroupKey(g.file, g.name)).Add(float64(missed))
 				}
 				evalTimestamp = evalTimestamp.Add((missed + 1) * g.interval)
+
+				useAlertsActiveAtOverrideFunc(g)
+
 				iter()
 			}
 		}
 	}
 }
 
+func useAlertsActiveAtOverrideFunc(g *Group) {
+	if g.alertsActiveAtOverrideFunc != nil {
+		for _, rule := range g.Rules() {
+			alertRule, ok := rule.(*AlertingRule)
+			if !ok {
+				continue
+			}
+			alertRule.ForEachActiveAlert(func(a *Alert) {
+				shouldOveride, activeAt := g.alertsActiveAtOverrideFunc(g, a, time.Now())
```

Comment:
Wondering if we should use time.Now or the time of the sample/alert/evaluation. If we only need time.Now(), this can be done in the downstream function.

Reply:
done